### ISSUE TITLE
Feature Rquest: Go offline

### ISSUE THREAD

**Issue by Beast141** [k9mail/k-9#1530]
Im not too sure if this is already there, but i cannot find it anywhere.

An option for going offline for a specific account should be available for each email account. When holding the email account down to open a menu there should be an option there for ease of use.
Until the user feels that the email account should come online, then he can manually switch it on again.

Any thoughts?

**Comment by gdt** [comment]
I agree that this would be nice, both to stop getting mail and to stop data usage.  Most other mail clients have an offline/online mode.   k-9 sort of does, but it's offline when the network is down.  So this would sort of be 'pretend the network is down'.

**Comment by Beast141** [comment]
Yes! especially for stopping data usage for a specific account.   This would be really beneficial!!
Hope this feature can be easily done.

**Comment by wwp** [comment]
Nice to have, indeed! Avoid editing all accounts or disabling network for this specific application by other means.

**Comment by steko** [comment]
Chiming in just to reinforce the idea that this switch, if implemented, should be at the account level (e.g. put work email offline while on vacation, keep personal email online at all times), and not at the application level.
